editors/emacs/Portfile

```diff
@@ -94,18 +94,19 @@ if {$subport eq $name || $subport eq "emacs-app"} {
 
 if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
     epoch           1
-    version         20200417
+    version         20200524
 
     fetch.type      git
-    git.url         http://git.savannah.gnu.org/r/emacs.git
-    git.branch      c36c5a3dedbb2e0349be1b6c3b7567ea7b594f1c
+    git.url         https://github.com/emacs-mirror/emacs.git
+    git.branch      cddb0079ff48fb839be742b157b709d1f1649709
 
     pre-configure {
         system -W ${worksrcpath} "sh ./autogen.sh"
     }
 
     configure.args-append  --with-json
-    depends_lib-append     port:jansson
+    depends_lib-append     port:jansson \
+        port:gmp
```

Comment:
please indent this at the same level as the other dep.

Reply:
done